website/docs/d/search_service.html.markdown

```diff
@@ -0,0 +1,85 @@
+---
+subcategory: "Search"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_search_service"
+description: |-
+  Manages a Search Service.
+---
+
+# azurerm_search_service
+
+Manages a Search Service.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+data "azurerm_search_service" "example" {
+  name                = "example-search-service"
+  resource_group_name = azurerm_resource_group.example.name
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The Name which should be used for this Search Service. Changing this forces a new Search Service to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Search Service should exist. Changing this forces a new Search Service to be created.
+
+## Attributes Reference
+
+In addition to the Arguments listed above - the following Attributes are exported:
+
+* `id` - The ID of the Search Service.
+
+* `primary_key` - The Primary Key used for Search Service Administration.
+
+* `secondary_key` - The Secondary Key used for Search Service Administration.
+
+* `query_keys` - A `query_keys` block as defined below.
+
+* `public_network_access_enabled` - Whether or not public network access is allowed for this resource. Defaults to `true`.
+
+* `partition_count` - The number of partitions which have been created.
+
+* `replica_count` - The number of replica's which have been created.
+
+-> **Note:** `partition_count` and `replica_count` can only be configured when using a `standard` sku.
```

Comment:
since this is a data source, this isn't configurable - so we can remove this (since the fields should get a default value either way)

```suggestion
```